GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -68,6 +86,10 @@ namespace gdjs {
           this._id === null ? '__default' : this._id
         );
         this._id = newID;
+        // Set the howl properties as soon as the sound is played and we have its ID.
+        this._howl.volume(this._volume, newID);
+        this._howl.loop(this._loop, newID);
+        this._howl.rate(this._rate, newID);
```

Comment:
Use gdjs.HowlerSoundManager.clampRate on the rate you're passing to this._howl.rate